<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="8" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="TestSuite" duration-ms="162625" started-at="2018-04-24T07:25:29Z" finished-at="2018-04-24T07:28:11Z">
    <groups>
    </groups>
    <test name="ChromeTest" duration-ms="133798" started-at="2018-04-24T07:25:29Z" finished-at="2018-04-24T07:27:43Z">
      <class name="Tests.TrackNumberTests">
        <test-method status="PASS" signature="Setup(java.lang.String)[pri:0, instance:Tests.TrackNumberTests@4d3167f4]" name="Setup" is-config="true" duration-ms="5009" started-at="2018-04-24T07:27:00Z" finished-at="2018-04-24T07:27:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="BlankEmailAddressTest()[pri:0, instance:Tests.TrackNumberTests@4d3167f4]" name="BlankEmailAddressTest" duration-ms="22318" started-at="2018-04-24T07:27:05Z" finished-at="2018-04-24T07:27:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BlankEmailAddressTest -->
        <test-method status="PASS" signature="BlankOrderNumberTest()[pri:0, instance:Tests.TrackNumberTests@4d3167f4]" name="BlankOrderNumberTest" duration-ms="6402" started-at="2018-04-24T07:27:27Z" finished-at="2018-04-24T07:27:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BlankOrderNumberTest -->
        <test-method status="FAIL" signature="BlankOrdernumberandEmailAddressTest()[pri:0, instance:Tests.TrackNumberTests@4d3167f4]" name="BlankOrdernumberandEmailAddressTest" duration-ms="8432" started-at="2018-04-24T07:27:33Z" finished-at="2018-04-24T07:27:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter a valid order numbr.] but found [Please enter a valid order number.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter a valid order numbr.] but found [Please enter a valid order number.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at pageModules.TrackOrderPage.VerifyOrderNumberMessage(TrackOrderPage.java:36)
at Tests.TrackNumberTests.BlankOrdernumberandEmailAddressTest(TrackNumberTests.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BlankOrdernumberandEmailAddressTest -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:Tests.TrackNumberTests@4d3167f4]" name="Teardown" is-config="true" duration-ms="823" started-at="2018-04-24T07:27:42Z" finished-at="2018-04-24T07:27:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- Tests.TrackNumberTests -->
      <class name="Tests.LoginTests">
        <test-method status="PASS" signature="Setup(java.lang.String)[pri:0, instance:Tests.LoginTests@76a3e297]" name="Setup" is-config="true" duration-ms="14112" started-at="2018-04-24T07:25:29Z" finished-at="2018-04-24T07:25:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="InvalidLoginDetails()[pri:0, instance:Tests.LoginTests@76a3e297]" name="InvalidLoginDetails" duration-ms="75789" started-at="2018-04-24T07:25:43Z" finished-at="2018-04-24T07:26:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InvalidLoginDetails -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:Tests.LoginTests@76a3e297]" name="Teardown" is-config="true" duration-ms="872" started-at="2018-04-24T07:26:59Z" finished-at="2018-04-24T07:27:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- Tests.LoginTests -->
    </test> <!-- ChromeTest -->
    <test name="FirefoxTest" duration-ms="162625" started-at="2018-04-24T07:25:29Z" finished-at="2018-04-24T07:28:11Z">
      <class name="Tests.TrackNumberTests">
        <test-method status="PASS" signature="Setup(java.lang.String)[pri:0, instance:Tests.TrackNumberTests@32eebfca]" name="Setup" is-config="true" duration-ms="15450" started-at="2018-04-24T07:27:18Z" finished-at="2018-04-24T07:27:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="BlankEmailAddressTest()[pri:0, instance:Tests.TrackNumberTests@32eebfca]" name="BlankEmailAddressTest" duration-ms="16770" started-at="2018-04-24T07:27:34Z" finished-at="2018-04-24T07:27:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BlankEmailAddressTest -->
        <test-method status="PASS" signature="BlankOrderNumberTest()[pri:0, instance:Tests.TrackNumberTests@32eebfca]" name="BlankOrderNumberTest" duration-ms="8695" started-at="2018-04-24T07:27:50Z" finished-at="2018-04-24T07:27:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BlankOrderNumberTest -->
        <test-method status="FAIL" signature="BlankOrdernumberandEmailAddressTest()[pri:0, instance:Tests.TrackNumberTests@32eebfca]" name="BlankOrdernumberandEmailAddressTest" duration-ms="8857" started-at="2018-04-24T07:27:59Z" finished-at="2018-04-24T07:28:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter a valid order numbr.] but found [Please enter a valid order number.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter a valid order numbr.] but found [Please enter a valid order number.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at pageModules.TrackOrderPage.VerifyOrderNumberMessage(TrackOrderPage.java:36)
at Tests.TrackNumberTests.BlankOrdernumberandEmailAddressTest(TrackNumberTests.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BlankOrdernumberandEmailAddressTest -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:Tests.TrackNumberTests@32eebfca]" name="Teardown" is-config="true" duration-ms="3351" started-at="2018-04-24T07:28:08Z" finished-at="2018-04-24T07:28:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- Tests.TrackNumberTests -->
      <class name="Tests.LoginTests">
        <test-method status="PASS" signature="Setup(java.lang.String)[pri:0, instance:Tests.LoginTests@28ac3dc3]" name="Setup" is-config="true" duration-ms="62457" started-at="2018-04-24T07:25:29Z" finished-at="2018-04-24T07:26:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="InvalidLoginDetails()[pri:0, instance:Tests.LoginTests@28ac3dc3]" name="InvalidLoginDetails" duration-ms="40772" started-at="2018-04-24T07:26:31Z" finished-at="2018-04-24T07:27:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InvalidLoginDetails -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:Tests.LoginTests@28ac3dc3]" name="Teardown" is-config="true" duration-ms="6243" started-at="2018-04-24T07:27:12Z" finished-at="2018-04-24T07:27:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- Tests.LoginTests -->
    </test> <!-- FirefoxTest -->
  </suite> <!-- TestSuite -->
</testng-results>
